README.md

```diff
@@ -305,55 +313,130 @@ Once you are done with the development, you can deploy the service using any of
 ### Deploying on Docker
 
 You can run the services that we developed above as a docker container. As Ballerina platform offers native support for running ballerina programs on containers, you just need to put the corresponding docker annotations on your service code. 
-Let's see how we can deploy the order_service we developed above on docker. When invoking this service make sure that the inventory_service is also up and running. 
+Let's see how we can deploy the order service and inventory we developed above on docker.
 
-- In our order_service, we need to import  `` import ballerinax/docker; `` and use the annotation `` @docker:Config `` as shown below to enable docker image generation during the build time. 
+- In our order service and inventory service, we need to import  `` import ballerinax/docker; `` and use the annotations `` @docker:Config `` , `` @docker:Expose `` as shown below to provide the basic docker image configurations and expose the listener port.
```

Comment:
Remove redundant back ticks.

And the whole import statement is not required when specifying the package